File: contribution/aquarium/utils/log_entry.rb
class LogEntry
  attr_accessor :name,:old_salary,:new_salary
  
  def initialize(name,old_salary,new_salary)
    @name = name
    @old_salary = old_salary
    @new_salary = new_salary
  end
  
  def to_s
    "LogEntry {name="+@name+", oldSalary="+@old_salary+", newSalary="+@new_salary+"}"
  end
end

File: contribution/aspectJ/org/softlang/utils/LogEntry.java
package org.softlang.utils;


public class LogEntry {

	private String name;

	private Double oldSalary;

	private Double newSalary;
	
	public LogEntry(String name, Double oldSalary, Double newSalary) {
		this.name = name;
		this.oldSalary = oldSalary;
		this.newSalary = newSalary;
	}

	public String getName() {
		return name;
	}

	public Double getOldSalary() {
		return oldSalary;
	}

	public Double getNewSalary() {
		return newSalary;
	}

	@Override
	public String toString() {
		return "LogEntry {name=\"" + name + "\", oldSalary=\"" + oldSalary
				+ "\", newSalary=\"" + newSalary + "\"}";
	}
}
